README.md

```diff
@@ -82,88 +82,11 @@ Conductor ready.
-
-## Environment for Development on this Project
-
-Assuming you have [installed the nix shell](https://nixos.wiki/wiki/Nix_Installation_Guide):
-
-```
-git clone git@github.com:holochain/holochain.git
-cd holochain
-nix-shell
-hc-merge-test
-```
-
-This will compile holochain and run all the tests.
-
-If you get an error while running `nix-shell` about ngrok having an unfree license, you can fix that by running,
-
-```
-mkdir -p ~/.config/nixpkgs/
-echo "{ allowUnfree = true; }" >> ~/.config/nixpkgs/config.nix
-```
-
-We have an all-in-one development environment including (among other things):
-
-- The correct version and sane environment variables of cargo/rust
-- Node for working with tryorama
-- Scaffolding, build and deployment scripts
-- Prebuilt binaries of core for various operating systems (soon)
-- Shared libs such as libsodium
-
-It is called [Holonix](https://github.com/holochain/holonix) and you should use it.
-
-It has plenty of documentation and functionality and can be used across Windows, Mac, and Linux.
-(Although Holochain itself currently only supports Linux.)
-It is based on the development tools provided by [NixOS](http://nixos.org/).
-
-It is suitable for use in hackathons and 'serious' development for a long-term,
-production grade development team.
-
-If you want to maintain your own development environment then we can only offer
-rough advice, because anything we say today could be out of date tomorrow:
-
-- Use a recent stable version of rust
-- Use node 12x+ for clientside work
-- Install any relevant shared libs like libsodium
-- Write your own scaffolding, build and development tools
-- Plan for dependency management as we ship new binaries
-
-### Advanced nix-shell usage
-
-#### Custom rust version
-
-The shell function in this repository takes a `rustVersion` argument that works in the following way:
-
-```shell
-$ nix-shell --arg rustVersion '{ track = "nightly"; version = "2021-07-01"; }' --run "rustc --version"
-Using /home/steveej/src/holo/holochain as target prefix...
-rustc 1.55.0-nightly (868c702d0 2021-06-30)
-```
-
-### Additional Documentation
-
-Build the holochain docs:
-```bash
-cargo doc --manifest-path=crates/holochain/Cargo.toml --open
-```
-
-## Contribute
-Holochain is an open source project.  We welcome all sorts of participation and are actively working on increasing surface area to accept it.  Please see our [contributing guidelines](/CONTRIBUTING.md) for our general practices and protocols on participating in the community, as well as specific expectations around things like code formatting, testing practices, continuous integration, etc.
-
-* Connect with us on our [forum](https://forum.holochain.org)
-
 ## License
- [![License: CAL 1.0](https://img.shields.io/badge/License-CAL%201.0-blue.svg)](https://github.com/holochain/cryptographic-autonomy-license)
 
-Copyright (C) 2019 - 2022, Holochain Foundation
+Copyright (C) 2019, Holochain Foundation
```

Comment:
do we really not want to extend the copyright to the current year?

Reply:
Copyright is implicit, we don't even need to state it explicitly. It doesn't need to be extended either. Only the year when the work was created should be stated. We can keep updating the year to indicate that we are creating new work if we want.
(see duration: https://www.copyright.gov/help/faq/faq-duration.html, copyright in general https://www.copyright.gov/help/faq/faq-general.html#mywork)